.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

We cut new releases weekly on monday or tuesday, so it would be easier on reviewers to batch these weekly as well and give plenty of time to review them, imo.
```suggestion
      interval: "weekly"
      day: "wednesday"
```

---

:+1: I committed your suggestion. Thanks.